examples/intro_to_inverse_problems.jl

```diff
@@ -45,6 +45,34 @@ observations = OneDimensionalTimeSeries(data_path, field_names=:b, normalize=ZSc
 # simulation to find optimal parameters by minimizing the discrepency between
 # the observations and the forward map.
 
+"""
+    extract_perfect_parameters(observations, Nensemble)
+
+Extract parameters from a batch of "perfect" observations.
+"""
+function extract_perfect_parameters(observations, Nensemble)
+    Nbatch = length(observations)
+    Qᵘ, Qᵇ, N², f = [zeros(Nensemble, Nbatch) for i = 1:4]
+
+    Nz = first(observations).grid.Nz
+    Hz = first(observations).grid.Hz
+    Lz = first(observations).grid.Lz
+    Δt = first(observations).metadata.parameters.Δt
+
+    for (j, obs) in enumerate(observations)
+        Qᵘ[:, j] .= obs.metadata.parameters.Qᵘ
+        Qᵇ[:, j] .= obs.metadata.parameters.Qᵇ
+        N²[:, j] .= obs.metadata.parameters.N²
```

Comment:
this line creates problems...
was stratification saved in `metadata.parameters`?

Reply:
I think line:

https://github.com/CliMA/OceanTurbulenceParameterEstimation.jl/blob/39fce1a53cf180d084d9042304c6601e4afd81c6/examples/intro_to_observations.jl#L38

creates trouble because a file is created *without* N² and then when `generate_synthetic_observations()` is called again it ends at that line because a file with that name exists. But I don't understand why that has happened...

Perhaps, when we call `generate_synthetic_observations()` we should provide `name` instead of using the default?